.gitignore

```diff
@@ -17,6 +17,7 @@ coverage
 artifacts
 screenshots
 bundles
+changed-files.json
```

Comment:
Why is changed-files here? This file is generated in CI only.

Reply:
The `changed-files.json` will be also created during the unit tests run.